ringvax/app.py

```diff
@@ -171,8 +172,12 @@ def app():
     sims = []
     with st.spinner("Running simulation..."):
         tic = time.perf_counter()
+
+        # initialize rngs
+        rngs = numpy.random.default_rng(seed).spawn(nsim)
```

Comment:
Nifty

Reply:
I've been waiting for a chance to use this

I spent a lot of time figuring out how this works for parallelizable simulations; see "Reproducible random number generation (RNG)" in https://cran.r-project.org/web/packages/furrr/furrr.pdf